lib/stimulus_reflex/channel.rb

```diff
@@ -18,6 +18,7 @@ def subscribed
 
   def receive(data)
     url = data["url"].to_s
+    selectors = data["selectors"].to_s
```

Comment:
I might consider splitting `selectors` here so the plural name always represents an array.